research-software-directory/RSD-as-a-service
# Hide masonry layout

# Hide masonry layout in software overview

Changes proposed in this pull request:
* The images currently used in the software pages are less suitable for masonry layout. Therefore we decided to remove masonry layout option in the first v2 release. 
* In addition, some refactoring is performed to share layout options between software and project overview pages because the options are currently identical. 
* The layout jump on software/project overview page when using list view (can be seen on dev) is reported by Jesus. This "layout jump" is addressed in this PR too.
* Tests are updated

How to test:
* `make start` to build app and generate test data
* navigate to software overview page and confirm that only two options are present: grid and list
* navigate to project overview page and confirm that only two options are present: grid and list
* navigate to organisation page and confirm that same options are present on organisation software and project pages
* select list view and navigate between software and project overview pages. During the changes the list view layout should be maintained without jumps or switches to some other layout option.

## Software overview
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/5d59b169-ca62-427a-a28f-cb12a40fb781)

## Project overview 
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/1e0be30e-c29c-43ed-ba77-f26a242db4c5)

## Organisation overview
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/53d1bf8d-8daa-4e02-98b6-6b0d946038bb)


PR Checklist:
